Library/AuthorizationHandler.ts

```diff
@@ -0,0 +1,52 @@
+﻿import http = require("http");
+import https = require("https");
+import azureCore = require("@azure/core-http");
+
+/**
+ * The automated token refresh will only start to happen at the
+ * expiration date minus the value of timeBetweenRefreshAttemptsInMs,
+ * which is by default 30 seconds.
+ */
+const timeBetweenRefreshAttemptsInMs = 30000;
+
+const applicationInsightsResource = "https://monitor.azure.com";
```

Comment:
since user is providing the token credential, is it a good idea to have them a option to provide "scope" too? If they donot provide the scope, we can use this default scope.

Reply:
Looking at other Azure SDKs implementations, scopes are never exposed and I don't believe we want the user to control which resource to authenticate, this is 100% managed by Breeze, if scope change we will need to update the SDK